### DISCUSSION TITLE
[Projects Beta] Project failed to load when interacting with iterations

### DISCUSSION THREAD

**Discussion by modeswitch** [community/community#10503] [Projects and Issues]
![Screenshot from 2022-01-24 12-58-08](https://user-images.githubusercontent.com/304186/150838219-ecacb2f5-dd1d-41de-b685-983c6ce2cef7.png)

**Comment by andreyrd** [comment]
![image](https://user-images.githubusercontent.com/2107076/150877625-fe85a89a-7c2c-437a-933a-17cc0faf5564.png)
Here is the relevant error in Console

**Reply by andreyrd** [reply]
I fixed it by adding a new upcoming iteration.
